src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -170,8 +171,24 @@ export default class TradeStore extends BaseStore {
         this.smart_chart        = this.root_store.modules.smart_chart;
         this.currency           = this.root_store.client.currency;
         const active_symbols    = await WS.activeSymbols();
-        if (!active_symbols.active_symbols || active_symbols.active_symbols.length === 0) {
+        if (active_symbols.error) {
             this.root_store.common.showError(localize('Trading is unavailable at this time.'));
+        } else if (!active_symbols.active_symbols || active_symbols.active_symbols.length === 0) {
+            const website_status = await BinarySocket.wait('website_status');
+            const residence_list = await WS.residenceList();
+
+            const clients_country_code = website_status.website_status.clients_country;
+            const clients_country_text =
+                (residence_list.residence_list.find(obj_country =>
+                    obj_country.value === clients_country_code) || {}).text;
+
+            this.root_store.common.showError(
+                localize('If you have an account, log in to continue.'),
+                (clients_country_text ? localize('Sorry, this app is unavailable in [_1].', clients_country_text) : localize('Sorry, this app is unavailable in your current location.')),
+                localize('Log in'),
+                redirectToLogin,
+                false,
+            );
```

Comment:
Can we move this from the trade-store to a helper function?
maybe: `showUnavailableLocationError(...)`

small thing: active_symbols.active_symbols.length === 0 --> !active_symbols.active_symbols.length